scripts/lib/CIME/SystemTests/pet.py

```diff
@@ -46,6 +46,7 @@ def _case_two_setup(self):
         # to the batch submission, things break. Setting MAX_TASKS_PER_NODE to half of
         # it original value prevents this.
         self._case.set_value("MAX_TASKS_PER_NODE", int(self._case.get_value("MAX_TASKS_PER_NODE") / 2))
+        self._case.set_value("MAX_MPITASKS_PER_NODE", int(self._case.get_value("MAX_MPITASKS_PER_NODE") / 2))
```

Comment:
PET tests are breaking on Cori-KNL and Theta after this change. I'm wondering if the first division (`MAX_TASKS_PER_NODE/2`) is still necessary. It looks like this was needed for skybridge. If it's still necessary, can there be a skybridge-specific change in config_machines.xml using a `$SHELL{...}` script? Skybridge has 16 cores, so the first 16x2 run would be on 2 nodes and the second 16x1 run would be on 1 node and mpiexec would not exceed procs-per-node?

Reply:
@amametjanov , sure.

Reply:
@amametjanov this will be fixed in the next CIME update.